### ISSUE TITLE
Add support for "start screen" plugins

### ISSUE THREAD

**Issue by vyfor** [vyfor/cord.nvim#89]
The ones in question are: [dashboard-nvim](https://github.com/nvimdev/dashboard-nvim), [vim-startify](https://github.com/mhinz/vim-startify), [mini.starter](https://github.com/echasnovski/mini.starter), [alpha-nvim](https://github.com/goolord/alpha-nvim), etc.
I'm curious whether it is a good idea to introduce a separate status for such plugins. My suggestion is that if any of these are active, we could display the [editor's logo](https://github.com/vyfor/cord.nvim/blob/master/assets/editor/neovim.png) on the large image, without displaying the small one. Open to suggestions.

**Comment by PCesteban** [comment]
I agree with your suggestion, @vyfor. Having one for each plugin is unnecessary. Having the editor's logo is a good solution since, in my case, the start screen displays "Editing a new file." I use alpha.

**Comment by vyfor** [comment]
Actually, now that I think about it, having a nice icon for dashboard plugins is not a bad idea. What do you all think?
![ptb discord com_channels_@me](https://github.com/user-attachments/assets/6978ddfd-2506-4bc5-9f7d-24d36401af45)

**Comment by vyfor** [comment]
This issue has been resolved in the new branch. I will be closing this as [v2](https://github.com/vyfor/cord.nvim/tree/client-server) has reached a stable state. Please consider migrating.
